hw/ip/aes/dv/err_injection_if/aes_err_injection.core

```diff
@@ -18,6 +18,8 @@ filesets:
       - fi_control_if.sv
       - fi_cipher_fsm_wrapper.sv
       - fi_cipher_if.sv
+      - fi_ctr_fsm_wrapper.sv
+      - fi_ctr_fsm_if.sv
```

Comment:
looks like you forgot to check in these 2 files

Reply:
yep that is correct. I just made the PR and went to bed. so only realized this morning.